src/config/dynamic-theme-fixes.config

```diff
@@ -3546,6 +3546,19 @@ CSS
 
 ================================
 
+cdn5.dcbstatic.com
+
+CSS
+#image942Img {
+    display: none !important;
+}
+li {
+    list-style-image: none !important;
+    list-style-type: circle !important;
+}
```

Comment:
What's the usage of this?

Reply:
Website using image of black dot in place of list marker that does not change with dark mode. These overrides disable the image and enable a circle to be used as the marker (website has marker disabled otherwise).